chainerx_cc/CMakeLists.txt

```diff
@@ -201,6 +209,19 @@ if(${CHAINERX_ENABLE_BLAS})
     endif()
 endif()
 
+# LAPACK
+if(${CHAINERX_ENABLE_LAPACK})
+    find_package(LAPACK QUIET)
+    if (${LAPACK_FOUND})
+        message(STATUS "Found LAPACK (library: ${LAPACK_LIBRARIES})")
+        message(STATUS "Defining CHAINERX_ENABLE_LAPACK variable.")
+        add_definitions(-DCHAINERX_ENABLE_LAPACK=1)
```

Comment:
It's defined here because this variable is also used in `chainerx_cc/chainerx/python/routines.cc`, not only for native device kernels, where lapack is linked.

Reply:
I see, thanks for the clarification.